src/functions.js

```diff
@@ -0,0 +1,127 @@
+import { updateLikes } from './likes.js';
+import { createComment, getComment } from './comment.js';
+import { countComments } from './counts.js';
+
+const mealsURL = 'https://www.themealdb.com/api/json/v1/1/filter.php?c=Seafood';
+const mealsContainer = document.querySelector('.meals-container');
+
+const getMealInstr = async (id) => {
+  const url = `https://www.themealdb.com/api/json/v1/1/lookup.php?i=${id}`;
+  const response = await fetch(url);
+  const data = await response.json();
+  const inst = data.meals[0].strInstructions;
+  return inst;
+};
+
+const getData = async () => {
+  const response = await fetch(mealsURL);
+  const data = await response.json();
+  return data;
+};
+
+const displayMeals = async () => {
+  mealsContainer.innerHTML = '';
+  const mealsOb = await getData();
+  mealsOb.meals.forEach((meal) => {
+    const mealDiv = document.createElement('div');
+    mealDiv.classList.add('meal-div');
+    // Call updateLikes() function to update the like count of the meal
+    updateLikes(meal, mealDiv);
+
+    mealDiv.innerHTML = `<div class="img-div" id=${meal.idMeal}>
+    <img src=${meal.strMealThumb} alt=${meal.strMeal} class="meal-img">
+  </div>
+  <a href="#" class="meal-title" data-id=${meal.idMeal}>${meal.strMeal}</a>
+  <div class="meals-details">
+  <i class="far fa-heart like-btn"></i>
+  <p class="like-count">Likes loading</p>
+  </div>
+  <button class="comment-btn" id="${meal.idMeal}">Comments</button>`;
+
+    // Append the list item element to the unordered list element
+    mealsContainer.appendChild(mealDiv);
+    // create the popup for comments
+
+    const commentButton = mealDiv.querySelector('.comment-btn');
+    commentButton.addEventListener('click', async (event) => {
+      event.preventDefault();
+      const mealId = event.target.id;
+      console.log(mealId);
```

Comment:
- [ ] You have done a great job to target the id for the comment popup. However, I suggest to consider removing the consoles. This will make our code professional and easy to maintain ☺️.

Reply:
Well resolved ✔️